website/docs/tutorials/08-cairo-1-support/01-project-initialization.md

```diff
@@ -34,6 +34,59 @@ This template contains:
     - Single test file with one test for function defined in `main.cairo`
 - `protostar.toml` containing information about the project
 
+## Cairo 1 modules
+
+In order to understand how to create cairo1 modules, we need to talk about `cairo_project.toml` and `lib.cairo` purpose.
+### Project defaults
+#### 1. `cairo_project.toml`
```

Comment:
Shouldn't cairo_project be autogenerated by Protostar? That's what's happening in Scarb

Reply:
Yes it is - that's why it's listed under "project defaults"

Reply:
I mean, we don't add to this file automatically if you meant that